integrationTests/factory/processComponents/processComponents_test.go

```diff
@@ -49,6 +50,12 @@ func TestProcessComponents_Close_ShouldWork(t *testing.T) {
 	require.Nil(t, err)
 	nodesShufflerOut, err := mainFactory.CreateNodesShuffleOut(managedCoreComponents.GenesisNodesSetup(), configs.GeneralConfig.EpochStartConfig, managedCoreComponents.ChanStopNodeProcess())
 	require.Nil(t, err)
+	nodesCoordinatorRegistryFactory, err := nodesCoord.NewNodesCoordinatorRegistryFactory(
```

Comment:
could you use the nodesCoordinatorRegistryFactory from managedBootstrapComponents instead?

Reply:
Very good suggestion, applied it !